packages/embed-react/README.md

```diff
@@ -83,6 +83,8 @@ The options for this integration are as follows.
 | `connectionOptions`       | `null`      | An optional set of options passed to a connection when processing a transaction (see https://docs.gr4vy.com/reference#operation/authorize-new-transaction)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `fullPageReturnUrl`       | `string`    | An optional return url that the user will be redirected to when embed is being used in in-app browsers or payments completed in popups where the merchant page has been closed (mobile)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `redirectMode`            | `fallback`  | `fallback`, `fullPage` - An optional configuration to change how redirects to payment providers are performed. `fallback` will attempt a popup before redirecting, `fullPage` will always redirect. This can be useful for testing purposes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| `redirectMode`            | `fallback`  | `fallback`, `fullPage` - An optional configuration to change how redirects to payment providers are performed. `fallback` will attempt a popup before redirecting, `fullPage` will always perform a full page redirect rather than using a popup. This can be useful for testing the fallback behaviour where a popup could not be used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```